src/components/card-detail/left-nav-section.tsx

```
@@ -25,6 +29,9 @@ const LeftNavSection = () => {
     error: getSlugError,
     isPending: isPendingSlug,
   } = useCardSlug(card_id);
+  useEffect(() => {
+    if (typeof window !== undefined) setHost(window.location.host);
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal
Unsafe fix: Compare the result of `typeof` with a valid type name


(lint/suspicious/useValidTypeof)